packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -890,7 +890,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
           width: widget.width,
           children: <Widget>[
             textField,
-            ..._initialMenu!,
+            ..._initialMenu!.map((Widget item) => controller.isOpen ? item : ExcludeFocus(child: item)),
```

Comment:
In order to prevent the widget tree from changing shape when the open state changes, you can use this:

```suggestion
            ..._initialMenu!.map((Widget item) => ExcludeFocus(excluding: !controller.isOpen, child: item)),
```